spec/models/photo_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Photo, type: :model do
+  describe '#uri' do
+    subject(:uri) { described_class.create(uri: 'http://www.ceva.ro').uri }
+
+    it { is_expected.to eq 'http://www.ceva.ro' }
+  end
```

Comment:
This is basically testing that the model has an `uri` attribute, I don't think this test provides much value, considering we have a `photo` factory which assumes the presence of an `uri` attribute.